nucleus/core/bootstrap/src/main/java/com/sun/enterprise/glassfish/bootstrap/GlassFishImpl.java

```diff
@@ -47,10 +47,13 @@
 import org.glassfish.embeddable.Deployer;
 import org.glassfish.embeddable.GlassFish;
 import org.glassfish.embeddable.GlassFishException;
+import org.glassfish.grizzly.threadpool.Threads;
 import org.glassfish.hk2.api.ServiceLocator;
 
 import java.util.Properties;
 
+import javax.swing.plaf.SliderUI;
```

Comment:
Why is swing being imported?

Reply:
My bad. Must have auto-completed wrong class and forgotten to remove it.